example/90-shoot.yaml

```
@@ -136,6 +136,9 @@ spec:
     #   - ReadonlyFilesystem
     #   - KernelDeadlock
     #   - DiskPressure
+    # sysctls: # optional, allows to specify kernel settings to override defaults
+    # - net.ipv4.tcp_wmem = "4096 131072 16777216"
+    # - net.ipv4.tcp_rmem = "4096 131072 16777216"
```

This is a map, right?

```suggestion
    #   net.ipv4.tcp_wmem: "4096 131072 16777216"
    #   net.ipv4.tcp_rmem: "4096 131072 16777216"
```

---

Thanks a lot. The weekend must have been too good ;-) .
Now, with https://github.com/gardener/gardener/commit/06bdf03f4ee1df5fbdaeb0f5e13b4e449a9d2a05 it should be better.